libs/editor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -1488,14 +1488,14 @@ public void onMediaUploadProgress(final String localMediaId, final float progres
 
     @Override
     public void onMediaUploadFailed(final String localMediaId, String errorType) {
```

Comment:
With this approach, we no longer require the `errorType` parameter and should remove it. 
```suggestion
    public void onMediaUploadFailed(final String localMediaId) {
```